ci/activemq.rb

```diff
@@ -0,0 +1,74 @@
+require './ci/common'
+
+def activemq_version
+  ENV['FLAVOR_VERSION'] || '5.11.1'
+end
+
+def activemq_rootdir
+  "#{ENV['INTEGRATIONS_DIR']}/activemq_#{activemq_version}"
+end
+
+namespace :ci do
+  namespace :activemq do |flavor|
+    task :before_install => ['ci:common:before_install']
+
+    task :install => ['ci:common:install'] do
+      unless Dir.exist? File.expand_path(activemq_rootdir)
+        sh %(curl -s -L\
+             -o $VOLATILE_DIR/activemq-#{activemq_version}.tar.gz\
+             http://archive.apache.org/dist/activemq/#{activemq_version}/apache-activemq-#{activemq_version}-bin.tar.gz)
```

Comment:
You can use `https://s3.amazonaws.com/dd-agent-tarball-mirror/apache-activemq-#{activemq_version}-bin.tar.gz`
Put the previous link in comment so that we can easily download the new tarball to our S3 bucket if needed.